Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -183,13 +193,16 @@ private void AddTriangleToSector(Vector3 sector, Vector3 point1, Vector3 point2,
 
                 int surfaceObjectIndex = SurfaceObjects.Count;
 
-                AddSurfaceObject(CreateSurfaceObject(
-                    mesh: nextSectorData.MeshObject,
-                    objectName: string.Format("SurfaceUnderstanding Mesh-{0}", surfaceObjectIndex),
-                    parentObject: transform,
-                    meshID: surfaceObjectIndex,
-                    drawVisualMeshesOverride: DrawProcessedMesh
-                    ));
+                SurfaceObject surfaceObject = CreateSurfaceObject(
+                  mesh: nextSectorData.MeshObject,
+                  objectName: string.Format("SurfaceUnderstanding Mesh-{0}", surfaceObjectIndex),
```

Comment:
Where do we use the object name?  Is it just for naming the object in the hierarchy of the scene?